packages/@aws-cdk/aws-route53/test/test.hosted-zone-provider.ts

```diff
@@ -1,47 +1,85 @@
 import { SynthUtils } from '@aws-cdk/assert';
 import cdk = require('@aws-cdk/core');
 import { Test } from 'nodeunit';
-import { HostedZone, HostedZoneAttributes } from '../lib';
+import { HostedZone } from '../lib';
 
 export = {
   'Hosted Zone Provider': {
-    'HostedZoneProvider will return context values if availble'(test: Test) {
+    'HostedZoneProvider will return context values if available'(test: Test) {
       // GIVEN
-      const stack = new cdk.Stack(undefined, 'TestStack', { env: { account: '12345', region: 'us-east-1' } });
-      const filter = {domainName: 'test.com'};
+      const stack = new cdk.Stack(undefined, 'TestStack', {
+        env: { account: '12345', region: 'us-east-1' }
+      });
+      const filter = { domainName: 'test.com' };
 
       HostedZone.fromLookup(stack, 'Ref', filter);
 
       const missing = SynthUtils.synthesize(stack).assembly.manifest.missing!;
       test.ok(missing && missing.length === 1);
 
+      const fakeZoneId = '11111111111111';
       const fakeZone = {
-        Id: "/hostedzone/11111111111111",
-        Name: "example.com.",
-        CallerReference: "TestLates-PublicZo-OESZPDFV7G6A",
+        Id: `/hostedzone/${fakeZoneId}`,
+        Name: 'example.com.',
+        CallerReference: 'TestLates-PublicZo-OESZPDFV7G6A',
```

Comment:
I'd prefer it if this code was completely reverted to demonstrate there is no regression.